Source/Communication/LogWatcher.cs

```diff
@@ -31,7 +31,14 @@ public LogWatcher()
 #if DEBUG && DEBUG_TEST_LOG
 			_logPath = @"Test\FastBuildLog.log";
 #else
-			_logPath = Path.Combine(Path.GetTempPath(), LogRelativePath);
+			string path = Path.GetTempPath();
```

Comment:
Prefer using `var` rather than concrete type
```suggestion
			var path = Path.GetTempPath();
```